ocaml/xapi/message_forwarding.ml

```diff
@@ -4379,5 +4379,24 @@ module Forward = functor(Local: Custom_actions.CUSTOM_ACTIONS) -> struct
              (fun () ->
                 do_op_on ~__context ~local_fn ~host
                   (fun session_id rpc -> Client.Cluster_host.disable rpc session_id self)))
+
+    let forget ~__context ~self =
+      info "Cluster_host.forget cluster_host:%s" (Ref.string_of self);
+      let cluster = Db.Cluster_host.get_cluster ~__context ~self in
+      let local_fn = Local.Cluster_host.disable ~self in
+      (* We need to ask another host that has a cluster host to mark it as dead.
+       * We might've run force destroy and this host would no longer have a cluster host
```

Comment:
Hmm, this host must have a cluster_host though, because that's the `self` parameter of this function

Reply:
Yes we need to find a live host somehow